src/sentry/api/endpoints/project_group_index.py

```diff
@@ -279,6 +279,7 @@ def put(self, request, project):
         search_fn = functools.partial(self._search, request, project)
         return update_groups(request, [project], project.organization_id, search_fn)
 
+    @attach_scenarios([bulk_remove_issues_scenario])
```

Comment:
Looks like this was accidentally removed here: https://github.com/getsentry/sentry/pull/11623/files#diff-25a63c81785c41e73ddc87f4058a70a4L800